building.sk

```diff
@@ -752,6 +752,7 @@ caves and cliffs update part 1:
 	deepslate lapis [lazuli] ore¦s = minecraft:deepslate_lapis_ore
 	deepslate diamond ore¦s = minecraft:deepslate_diamond_ore
 	
-	[any] stone ore = any ore
+	[any] stone ore = gold ore, iron ore, coal ore, lapis ore, diamond ore, redstone ore, emerald ore
```

Comment:
You should be okay to specify `any ore, copper ore` here as then you don't have to repeat all of the other stone ores :)